File: output/Carrera.java
public class Carrera{
	private int idCarrera;
	private String claveCarrera;
	private String nombreCarrera;
	private int idPlanEstudio;

	public int getIdCarrera(){
		return this.idCarrera;
	}
	public void setIdCarrera(int idCarrera){
		this.idCarrera=idCarrera;
	}
	public String getClaveCarrera(){
		return this.claveCarrera;
	}
	public void setClaveCarrera(String claveCarrera){
		this.claveCarrera=claveCarrera;
	}
	public String getNombreCarrera(){
		return this.nombreCarrera;
	}
	public void setNombreCarrera(String nombreCarrera){
		this.nombreCarrera=nombreCarrera;
	}
	public int getIdPlanEstudio(){
		return this.idPlanEstudio;
	}
	public void setIdPlanEstudio(int idPlanEstudio){
		this.idPlanEstudio=idPlanEstudio;
	}
}

File: output/Carrera.php
<?php
class Carrera{
	private  $idCarrera;
	private  $claveCarrera;
	private  $nombreCarrera;
	private  $idPlanEstudio;

	public function getIdCarrera(){
		return $this->idCarrera;
	}
	public  function setIdCarrera($idCarrera){
		$this->idCarrera=$idCarrera;
	}
	public function getClaveCarrera(){
		return $this->claveCarrera;
	}
	public  function setClaveCarrera($claveCarrera){
		$this->claveCarrera=$claveCarrera;
	}
	public function getNombreCarrera(){
		return $this->nombreCarrera;
	}
	public  function setNombreCarrera($nombreCarrera){
		$this->nombreCarrera=$nombreCarrera;
	}
	public function getIdPlanEstudio(){
		return $this->idPlanEstudio;
	}
	public  function setIdPlanEstudio($idPlanEstudio){
		$this->idPlanEstudio=$idPlanEstudio;
	}
}
?>